libr/anal/p/anal_evm_cs.c

```diff
@@ -256,6 +345,29 @@ static char *get_reg_profile(RAnal *anal) {
 	return (p && *p)? strdup (p): NULL;
 }
 
+static bool evm_anal_init (void *user) {
```

Comment:
```suggestion
static int evm_anal_init(void *user) {
```

those functions returns int right now. imho it will be better to use bool for init and void for fini, but better stick to what's in master right now and we will massage it during the arch refactoring